docs/stdlib/json.rst

```diff
@@ -75,10 +76,11 @@ Anything in EdgeDB can be cast into :eql:type:`json`:
     db> select <json>cal::to_local_date(datetime_current(), 'UTC');
     {'"2019-04-02"'}
 
-Any :eql:type:`Object` can be cast into :eql:type:`json`. This
-produces the same JSON value as the JSON serialization of that object.
-That is, the result is the same as the output of :eql:stmt:`select
-expression <select>` in *JSON mode*, including the type shape.
+Additionally, any :eql:type:`Object` in EdgeDB can be casted as a
+:eql:type:`json` type. This produces the same JSON value as the
+JSON-serialized result of that said object. Furthermore, this result will
+be the same as the output of a :eql:stmt:`select expression <select>` in
+*JSOM mode*, including the shape of that type:
```

Comment:
```suggestion
*JSON mode*, including the shape of that type:
```

In addition to this, is there a page that would be appropriate to link to for "JSON Mode?" If not, maybe we could describe it here?

Reply:
I'm not sure what "JSON mode" really describes here, I take it that it involves using `SELECT` onto a `<bool>` cast type

Reply:
Could you provide some context on "JSON mode," @vpetrovykh?